extensions/ql-vscode/src/queryserver-client.ts

```diff
@@ -259,6 +259,14 @@ export function findQueryLogFile(resultPath: string): string {
   return path.join(resultPath, 'query.log');
 }
 
-export function findQueryStructLogFile(resultPath: string): string {
+export function findQueryEvalLogFile(resultPath: string): string {
   return path.join(resultPath, 'evaluator-log.jsonl');
 }
+
+export function findQueryEvalLogSummaryFile(resultPath: string): string {
+  return path.join(resultPath, 'evaluator-log.jsonl.summary');
```

Comment:
Is this a valid json or jsonl file? If so, it should probably use the proper file extension. Maybe:

```suggestion
  return path.join(resultPath, 'evaluator-log.summary.jsonl');
```

Reply:
It's not — it's a generated text file summarizing the contents of the jsonl file.

Reply:
OK...then maybe avoid the `jsonl` component entirely?

Reply:
Just calling it `.summary` seems perfectly reasonable to me, yep!